docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,51 @@
+## Using RepoSense with Netlify 
+                                 
+Netlify's continuous web deployment platform enables you to deploy RepoSense online, analyzes your target repository(s) and generates the git contribution report in real time for free. The deployment can be synchronized to the updates on your target repository(s) and you can perform everything through the browser without downloading any files.
+
+1. Fork the RepoSense repository using this [link](https://github.com/repoSense/RepoSense/fork).
+1. Follow this [guide](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) to set up Netlify in your forked repository. You will need to use the following in **Step 5** of the guide:
```

Comment:
can use this url for the Step 5
https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/#step-5-configure-your-settings

~~of the guide~~ can be removed

Reply:
Done